src/menus/tripmenu.cpp

```diff
@@ -2990,7 +2990,7 @@ void credits_menu(void) {
         grid3->printf(205, 110, "Code & Project management:\nTeemu J. Takanen");
         grid3->printf(kohta2, 140, "Special Thanks:");
         grid3->printf(45, 160, "Mikko Kinnunen\nNiko Salminen\nPekka Pulli");
-        grid3->printf(205, 160, "Antti Lehtoranta\nJuha Rytk�nen\nJoonas Joensuu");
+        grid3->printf(205, 160, "Antti Lehtoranta\nJuha Rytk\xf6nen\nJoonas Joensuu");
```

Comment:
Hmm, this doesn't work on Windows.

![kuva](https://user-images.githubusercontent.com/24453333/88198091-00ba1f80-cc4c-11ea-93df-512789407e11.png)

Reply:
I may have mistranscribed the character, then...

Reply:
Okay, corrected to use 0x94:

```
$ xxd src/menus/tripmenu.cpp | grep -C2 Juha
000132a0: 2d3e 7072 696e 7466 2832 3035 2c20 3136  ->printf(205, 16
000132b0: 302c 2022 416e 7474 6920 4c65 6874 6f72  0, "Antti Lehtor
000132c0: 616e 7461 5c6e 4a75 6861 2052 7974 6b94  anta\nJuha Rytk.  <-- see here
000132d0: 6e65 6e5c 6e4a 6f6f 6e61 7320 4a6f 656e  nen\nJoonas Joen
000132e0: 7375 7522 293b 0a0a 0a0a 2020 2020 2020  suu");....
```